includes/compat/class-customer.php

```diff
@@ -274,28 +276,33 @@ public function update_user_meta( $check, $object_id, $meta_key, $meta_value, $p
 		if ( is_array( $addresses ) && ! empty( $addresses ) ) {
```

Comment:
I know this isn't part of your changes, but while we're here...

```suggestion
		if ( is_array( $addresses ) && ! empty( $addresses[0] ) ) {
```

Because `$address[0]` is used on the next line.